src/plugins.star

```diff
@@ -0,0 +1,78 @@
+common = import_module("github.com/kurtosis-tech/autogpt-package/src/common.star")
```

Comment:
can we keep the `plugins.star` in root? lets move common.star to root as well(to make it standard)

We link to it here https://github.com/Significant-Gravitas/Auto-GPT-Plugins#first-party-how-to and other places and I don't want to change things too much

Reply:
another benefit of keeping it in root is discovery